Doppler.ContactPolicies.Data.Access/Entities/ContactPoliciesSettings.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Doppler.ContactPolicies.Data.Access.Entities
+{
+    public sealed class ContactPoliciesSettings
+    {
+
+        public User User { get; set; }
+        public bool Enabled { get; set; }
+        public bool Active { get; set; }
+        public int Amount { get; set; }
+        public int Interval { get; set; }
```

Comment:
What do you think about using more explicit names? 

For example:

```suggestion
        public int EmailsAmountByInterval { get; set; }
        public int IntervalInDays { get; set; }
```

Reply:
not a problem to me, i just put it in like this by convension with the names in the data base